recipe/meta.yaml

```diff
@@ -10,8 +10,6 @@ package:
 source:
   git_url: https://github.com/tudat-team/tudatpy.git
   git_rev: {{ git_rev }}
-  patches:
-    - use_sysconfig_for_python_paths.patch
```

Comment:
Is Python 3.12 passing without this now, or has this been added to the tudatpy source?

Reply:
I increased the minimum required CMake version for tudatpy to be able to use the "new" method for finding Python, so we don't need that part of the YACMASetup script anymore

Reply:
When you say 3.12 I assume you refer to the one that is used to generate the macOS distributions, right? Because we only support 3.9 to 3.11 right now (until I figure out why pydantic does not support 3.12 or why are we forcing the use of a version that doesn't)

Reply:
Sorry, I'm not up to date on what you're working towards with CMake at the moment, but I added that patch so that Python 3.12 was supported by tudatpy and found using YACMA.

Feel free to provide links and a summary of what you're fixing.